src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/ECCharacteristic2Curve.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// A prime elliptical curve.
+    /// </summary>
+    public sealed class ECCharacteristic2Curve : ECExplicitCurve
+    {
+        public byte[] M;
```

Comment:
All. I'm open for naming this something different. If Windows ever adds char2 support, hopefully it uses this binary format as well. 

Openssl uses the prime 'p' parameter as the char2 'm' parameter in its APIs.

Example:  K-163 has the values in openssl like this:
	  0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,	/* p */
	  0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
	  0xC9,
h=2 in this case.

From my sources this curve has explicit polynomial of:
m=163, f(z)=z^163 + z^7 + z^6 + z^3 + 1
which the above binary equals

Also see https://en.wikisource.org/wiki/NIST_Koblitz_Curves_Parameters

Openssl says this: "EC_GROUP_set_curve_GF2m sets the equivalent curve parameters for a curve over F2^m. In this case p represents the irreducible polynomial - each bit represents a term in the polynomial. Therefore there will either be three or five bits set dependent on whether the polynomial is a trinomial or a pentanomial. EC_group_get_curve_GF2m obtains the previously set curve parameters."